packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -221,6 +221,7 @@ const Combobox = React.forwardRef<HTMLInputElement, ComboboxProps>(
           {labelText}
         </Label>
         <InputBox
+          autocomplete={autocomplete}
```

Comment:
Why does `InputBox` need to know about autocomplete?

Reply:
It made sense to me to pass it like the other props, but maybe I just need the autocomplete type to be available. Thinking about it more now, it probably doesn't need to be passed since its not doing anything to `InputBox`.